Task7/src/index.js

```diff
@@ -0,0 +1,209 @@
+        return this;
+    };
+
+    static addClass(selector, className){
+        let elements = document.querySelectorAll(selector);
+        for(let i = 0; i < elements.length; i++ ) {
+            elements[i].classList.add(className);
+        }
+    };
+
+    removeClass(className){
+        jQuery.removeClass(this._string, className);
+        return this;
+    }
+
+    static removeClass(selector, className){
+        let elements = document.querySelectorAll(selector);
+        for(let i = 0; i < elements.length; i++ ) {
+            elements[i].classList.remove(className);
+        }
+    };
+
+    append(strHtml){
+        jQuery.append(this._string, strHtml);
+        return this;
+    }
+
+    static append(selector, strHtml){
+        let elements = document.querySelectorAll(selector);
+        for(let i = 0; i < elements.length; i++ ) {
+            elements[i].insertAdjacentHTML('afterbegin', strHtml);
+        }        
+    };
+
+    remove(){
+        jQuery.remove(this._string);
+        return this;
+    }
+
+    static remove(selector){
+        let elements = document.querySelectorAll(selector);
+        for(let i = 0; i < elements.length; i++ ){
+            elements[i].parentNode.removeChild(elements[i]);
+        }        
+    };
+
+    text(){
+        return jQuery.text(this._string);
+    }
+
+    static text(node){
+        let element = document.querySelector(node);
+        if (element == null){
+            return null;
+        }
+        return element.innerText;        
+    };
+
+    attr(attributeName, value){
+        return jQuery.attr(this._string, attributeName, value);
+    };
+
+    //Get the value of an attribute for the first element in the set of matched elements
+    //value != null set one or more attributes for every matched element.
+    static attr(selector, attributeName, value){
+        let element = document.querySelector(selector);
+        if (element == null){
+            return null;
+        }
+
+        if (value){
+            element.setAtttibute(attributeName, value);
+        }
+        else{
+            return element.getAttribute(attributeName);            
+        }
+    };
+
+    children(selector){
+        return new jQuery( jQuery.children(this._string, selector));
+    }
+
+    //Get the children of each element in the set of matched elements, optionally filtered by a selector.
+    static children(node, selector){
+        let element = document.querySelector(node);
+        if (element == null){
+            return null;
+        }
+        let result = [];
+        if (!selector)
+            return element.childNodes;
+
+        for(let i = 0; i < element.childNodes.length; i++ ){            
+            if (element.childNodes[i].nodeType == 1 && 
+                (element.childNodes[i].matches == "undefined"
+                ? element.childNodes[i].matches(selector)
+                :element.childNodes[i].msMatchesSelector(selector)
```

Comment:
better to move into separate function and replace with `matches(element.childNodes[i])`